Includes/Rosetta/Enums/TriggerEnums.hpp

```diff
@@ -18,9 +18,13 @@ enum class TriggerType
     PLAY_CARD,   //!< The effect will be triggered when a player plays a card.
     CAST_SPELL,  //!< The effect will be triggered when a player plays a Spell
                  //!< card.
+    AFTER_CAST,  //!< The effect will be triggered after a spell is played.
     HEAL,        //!< The effect will be triggered when characters are healed.
     ATTACK,      //!< The effect will be triggered when characters attack.
     SUMMON,  //!< The effect will be triggered whenever a minion is summoned.
+    PREDAMAGE,  //!< Effect will be triggered when a character gets predamage.
+                //!< This event happens just before the character is actually
+                //!< damaged.
```

Comment:
please, align the indentation of the comments!

Reply:
Already aligned comment by clang-format.

Reply:
oh, i just mean, pad a `space` to the right!

Reply:
I don’t understand why I pad a space to the right. It is aligned using clang-format style.

Reply:
oh my mistake :( haha...